infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/brokerphases/BrokerEnvironmentFactory.java

```diff
@@ -87,14 +91,20 @@ public BrokerEnvironmentFactory(
       String artifactsBrokerImage,
       String metadataBrokerImage,
       String pluginRegistryUrl,
+      String pluginRegistryInternalUrl,
       CertificateProvisioner certProvisioner) {
     this.cheWebsocketEndpoint = cheWebsocketEndpoint;
     this.brokerPullPolicy = brokerPullPolicy;
     this.authEnableEnvVarProvider = authEnableEnvVarProvider;
     this.machineTokenEnvVarProvider = machineTokenEnvVarProvider;
     this.artifactsBrokerImage = artifactsBrokerImage;
     this.metadataBrokerImage = metadataBrokerImage;
+    if (!isNullOrEmpty(pluginRegistryInternalUrl)) {
+      pluginRegistryUrl = pluginRegistryInternalUrl;
+    }
+
     this.pluginRegistryUrl = pluginRegistryUrl;
+    LOGGER.info("Plugin registry url: {}", this.pluginRegistryUrl);
```

Comment:
do we really need this on info level? Do we need it to log at all?

Reply:
+1 to remove it